server/src/test/python/cromwell-az-e2e-test/cromwell_az_e2e_test/az_e2e.py

```diff
@@ -0,0 +1,229 @@
+import requests
+import os
+import json
+import random
+import string
+import uuid
+import time
+from collections import deque
+
+# NOTE: POETRY prefix is outlined per documentation here: https://python-poetry.org/docs/configuration#using-environment-variables
+bearer_token = os.environ['POETRY_BEARER_TOKEN']
+bee_name = os.environ['POETRY_BEE_NAME']
+billing_project_name = os.environ['POETRY_BILLING_PROJECT_NAME']
+
+rawls_url = f"https://rawls.{bee_name}.bee.envs-terra.bio"
+leo_url = f"https://leonardo.{bee_name}.bee.envs-terra.bio"
+
+def output_message(msg):
+    current_time = time.strftime("%H:%M:%S", time.localtime())
+    print(f'{current_time} - {msg}')
+
+def handle_failed_request(response, msg, status_code=200):
+    if(response.status_code != status_code):
+        raise Exception(f'{response.status_code} - {msg}\n{response.text}')
+
+def create_workspace():
+   rawls_api_call = f"{rawls_url}/api/workspaces"
+   request_body= {
+      "namespace": billing_project_name, # Billing project name
+      "name": f"api-workspace-{''.join(random.choices(string.ascii_lowercase, k=5))}", # workspace name
+      "attributes": {}}
+   
+   create_workspace_response = requests.post(url=rawls_api_call, 
+                                             json=request_body, 
+                                             headers={"Authorization": f"Bearer {bearer_token}"}
+   ).json()
+
+   create_workspace_data = json.loads(json.dumps(create_workspace_response))
+   workspace_id = create_workspace_data['workspaceId']
+
+   output_message(f"Enabling CBAS for workspace {workspace_id}")
+   activate_cbas_request = f"{leo_url}/api/apps/v2/{workspace_id}/terra-app-{str(uuid.uuid4())}"
+   cbas_request_body = {
```

Comment:
This is correct right now, just noting that the switch to multi-user workflows is imminent and we may want to switch to `CROMWELL_RUNNER` very soon.

Reply:
What would the update entail at a high-level (new Leo endpoint, different format for app_url response, etc.)?